### PR TITLE
release notes draft

### PR DESCRIPTION
**Pull request by michaelcretzman**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md [hunk 1]
```diff
@@ -16,6 +16,18 @@ For Harness on-prem releases, see [Harness Self-Managed Enterprise Edition Relea
 
 If you don't see a new feature or enhancement in your Harness account, it might be behind a Feature Flag. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
 
+### March 7, 2023, version 786xx
```
Comment (aimurphy) [inline docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md:19]: don't forget the date in the metadata for these.

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -13,6 +13,101 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+  This functionality is behind a feature flag: `OPTIMIZED_GIT_FETCH_FILES`.
+
+  As an alternative to username and password authentication, you can now use [Jira PAT authentication](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html) in a Harness Jira Connector.
+
+### Fixed issues
+
+- The **Default Settings** category is selected when a user selects the **GitOps** category. (CDS-53975)
+
+  The **Default Settings** and **Gitops** categories should not both be selected when the **Gitops** category is selected.
```
Comment (aimurphy) [inline release-notes/continuous-delivery.md:34]: capitalization of GitOps is different

File: release-notes/continuous-delivery.md [hunk 3]
```diff
@@ -13,6 +13,101 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+  
+  Now, if there are empty values for values YAML **File Path**, these values YAML settings are omitted from the service UI.
+- The **Manual Intervention** tab was not displayed for the ServiceNow Create or Update steps. (CDS-53467, ZD-38687)
+  
+  The **Manual Intervention** tab is used to select a failure strategy when the acceptance criteria in a ServiceNow [Create](https://developer.harness.io/docs/continuous-delivery/cd-advanced/ticketing-systems-category/create-service-now-tickets-in-cd-stages/) or [Update](https://developer.harness.io/docs/continuous-delivery/cd-advanced/ticketing-systems-category/update-service-now-tickets-in-cd-stages) step is not met. 
+  
+  The **Manual Intervention** tab was not visible, but this has been fixed and the tab now displays during pipeline execution.
+
+  ![Manual Intervention tab](static/aaff20d556d48292426c819a4e8542f31f202204a10fbf6c1b6b9fe4f8447831.png)  
+- The error message for unsupported connector types in the Kubernetes **Infrastructure Definition** was not clear. (CDS-53458)
```
Comment (aimurphy) [inline release-notes/continuous-delivery.md:56]: Noticed that some are in past tense are some are in present tense. My personal preference is past tense for the description of the problem.

File: release-notes/continuous-delivery.md [hunk 4]
```diff
@@ -13,6 +13,101 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+  
+  When a user selected an incorrect Harness connector type, the error message was not clear why.
+
+  Now the error message has been improved.
+
+  ![incorrect Harness connector type message](static/f1c6d8873ddc32da338ba1f86ea630338a8567cfc41b37eda9a519945313cc84.png)  
+ 
+- The environment's **Service Overrides** were not operating additively. (CDS-53373)
+  
+  You can override specific service settings using the [Service Overrides settings](https://developer.harness.io/docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview#service-overrides) in an environment. Whenever a specific service is used with the environment, the environment's **Service Overrides** settings override the service's setting.
```
Comment (aimurphy) [inline release-notes/continuous-delivery.md:68]: Lol I don't think you can do much about this, but it sounds like a tongue twister... sally sells seashells by the seashore.

File: release-notes/continuous-delivery.md [hunk 5]
```diff
@@ -13,6 +13,101 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+  ![Service Overrides settings](static/88805f0c3a1feca13b5437edbd6c7574e8f540a6e9ffe07f760a450546c93c41.png)
+
+  For values YAML files, the override operation is additive. If the **Service Overrides** settings contain values YAML settings that are not in the service, then they are added to the service values YAML file.
```
Comment (aimurphy) [inline release-notes/continuous-delivery.md:72]:
```suggestion
  For `values` YAML files, the override operation is additive. If the **Service Overrides** settings contain `values` YAML settings that are not in the service, then they are added to the service's `values` YAML file.
```

Suggestion to use code notation for file name for readability.

File: release-notes/continuous-delivery.md [hunk 6]
```diff
@@ -13,6 +13,101 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+  
+  The override was not operating additively. It was completely overriding the service settings. This has been fixed. The override operation is now performing additively.
+
+  :::note
+  
+  Config files are a black box that can contain multiple formats and content, such as YAML, JSON, plain text, etc. Consequently, they cannot be overridden like Values YAML files. Variables cannot be partially overridden either. They are completely replaced.
```
Comment (aimurphy) [inline release-notes/continuous-delivery.md:78]:
```suggestion
  Config files are a black box that can contain multiple formats and content, such as YAML, JSON, plain text, etc. Consequently, they cannot be overridden like `values` YAML files. Variables cannot be partially overridden either. They are completely replaced.
```

File: release-notes/continuous-delivery.md [hunk 7]
```diff
@@ -13,6 +13,101 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+- A reused [webhook trigger](https://developer.harness.io/docs/platform/triggers/trigger-deployments-using-custom-triggers/) identifier results in a reused last activation time. (CDS-53107)
+  
+  When a webhook trigger is deleted and then a new trigger is created with same identifier, the last activation time displayed for the new trigger is incorrect.
+  
+  The trigger event history for the first trigger was not deleted. This resulted in showing the stale information when a new trigger with the same identifier was used.
+  
+  Now, once a trigger delete succeeds, the event history for the trigger is deleted as well. This cleans up all the information related to the deleted trigger.
+- Empty string inputs in triggers were treated as objects.(CDS-53078)
+  
+  Empty string inputs caused errors when a runtime input was expecting an object.
```
Comment (aimurphy) [inline release-notes/continuous-delivery.md:100]: Some of these feel a bit redundant between the initial bullet and the next line describing the issue again. Maybe a consideration for future, but in some cases it seems like the first two lines could be combined. Ex: Empty string inputs in triggers caused errors when runtime input expected an object and treated the empty string as an object.

**Review by aimurphy — DISMISSED** [review]
approv w assumption you'll consider my comments

File: release-notes/continuous-delivery.md (hunk 5 above)
Comment (michaelcretzman) [reply release-notes/continuous-delivery.md:72]: it's not code. it's a type of file. I would have to do the same for manifest, config file, etc.

File: release-notes/continuous-delivery.md (hunk 7 above)
Comment (michaelcretzman) [reply release-notes/continuous-delivery.md:100]: good point. will revise.

**Review by deepakbolangady — APPROVED** [review]
Updated the date in today's CD release notes. Also updated the metadata and resolved a merge conflict with PL release notes.

**Pull request merged**